runtime/Go/antlr/go.mod

```diff
@@ -1,3 +1,5 @@
 module github.com/antlr/antlr4/runtime/Go/antlr
 
 go 1.18
+
+require golang.org/x/exp v0.0.0-20220722155223-a9213eeb770e
```

Comment:
You might have fixed later...do we want to require exp stuff?

Reply:
Yes - I added this and go.sum as it is required, though go works it out. We do need to require it, but this package is part of the runtime that comes with go.